src/app/auth/callback/kakao/page.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+
+import { useQuery } from '@tanstack/react-query';
+import { useSearchParams } from 'next/navigation';
+import { useRouter } from 'next/navigation';
+import { useEffect } from 'react';
+
+import publicApi from '~/api/config/publicApi';
+import { AuthResponse } from '~/types/auth';
+import { generateCookiesKeyValues } from '~/utils/auth/tokenHandlers';
+
+const KakaoCallbackPage = () => {
+  const searchParams = useSearchParams();
+  const router = useRouter();
+
+  const code = searchParams.get('code');
+  const { data } = useQuery<AuthResponse>(['login'], async () => {
+    const host = window.location.protocol + '//' + window.location.host;
```

Comment:
아래와 같이 축약할 수 있을 것 같습니다!
```suggestion
    const host = window.location.origin;
```

Reply:
오 꿀팁 감사합니다!